lib/kapnismology/terminal.rb

```diff
@@ -1,20 +1,26 @@
 module Kapnismology
   class Terminal
-
     def self.green(msg)
-      "\e[32m\e[1m#{msg}\e[0m"
+      colorize("\e[32m", msg)
     end
 
     def self.red(msg)
-      "\e[31m\e[1m#{msg}\e[0m"
+      colorize("\e[31m", msg)
     end
 
     def self.yellow(msg)
-      "\e[33m\e[1m#{msg}\e[0m"
+      colorize("\e[33m", msg)
     end
 
     def self.bold(msg)
-      "\e[1m#{msg}\e[0m"
+      colorize('', msg)
+    end
+
+    private
+    # \e[1m adds bold font, \e[0m resets all styles
```

Comment:
[nits] this comment about the codes should be within the method because it refers to the logic itself